tests/test_qdrant_client.py

```diff
@@ -2134,13 +2134,21 @@ def test_create_payload_index(prefer_grpc):
     client = init_remote(prefer_grpc=prefer_grpc)
     initialize_fixture_collection(client, COLLECTION_NAME, vectors_config={})
 
-    client.create_payload_index(COLLECTION_NAME, "keyword", models.PayloadSchemaType.KEYWORD)
-    client.create_payload_index(COLLECTION_NAME, "integer", models.PayloadSchemaType.INTEGER)
-    client.create_payload_index(COLLECTION_NAME, "float", models.PayloadSchemaType.FLOAT)
-    client.create_payload_index(COLLECTION_NAME, "geo", models.PayloadSchemaType.GEO)
-    client.create_payload_index(COLLECTION_NAME, "text", models.PayloadSchemaType.TEXT)
-    client.create_payload_index(COLLECTION_NAME, "bool", models.PayloadSchemaType.BOOL)
-    client.create_payload_index(COLLECTION_NAME, "datetime", models.PayloadSchemaType.DATETIME)
+    client.create_payload_index(
+        COLLECTION_NAME, "keyword", models.PayloadSchemaType.KEYWORD, wait=True
```

Comment:
I assume all these `wait=true` are added now because of the new non-blocking payload index creation 👍